### ISSUE TITLE
grpo没有办法在output中保存images的文件夹。

### ISSUE THREAD

**Issue by drewqueaz-ctrl** [modelscope/ms-swift#5493]
echo 'start.....'

CUDA_VISIBLE_DEVICES=0,1,2,3 \
NPROC_PER_NODE=4 \
swift rlhf \
    --rlhf_type grpo \
    --model Qwen/Qwen2.5-7B-Instruct \
    --train_type full \
    --reward_funcs accuracy \
    --dataset AI-MO/NuminaMath-TIR#10000 \
    --torch_dtype bfloat16 \
    --use_vllm true \
    --vllm_mode colocate \
    --vllm_gpu_memory_utilization 0.5 \
    --vllm_max_model_len 8000 \
    --vllm_tensor_parallel_size 4 \
    --num_train_epochs 1 \
    --max_length 8000 \
    --per_device_train_batch_size 4 \
    --per_device_eval_batch_size 4 \
    --gradient_accumulation_steps 1 \
    --eval_steps 200 \
    --save_steps 50 \
    --learning_rate 1e-6 \
    --save_total_limit 20 \
    --logging_steps 5 \
    --output_dir output \
    --warmup_ratio 0.05 \
    --dataloader_num_workers 4 \
    --max_completion_length 2048 \
    --num_generations 8 \
    --deepspeed zero3 \
    --temperature 1.0 \
    --top_p 1.0 \
    --top_k 80 \
    --log_completions true \
    --async_generate false \
    --offload_optimizer true \
    --offload_model true \
    --sleep_level 1 \
    --multi_turn_scheduler math_tip_trick_multi_turn \
    --max_turns 3

这个是我的参数，为什么现在一样的设置下原本会在output中保存images，现在不会了，是我的什么参数没有设置对吗？

**Comment by hjh0119** [comment]
Did you actually run 50 training steps since you set --save_steps 50?

**Comment by Zzzzzxl-git** [comment]
老哥解决了吗？我跑了 grpo ，output 文件夹里也是没有 images

**Comment by z0o0ey** [comment]
please retry with latest swift and vllm, feel free to reopen or create a new issue if it the problem persists.
